src/components/TextInput/PTextInput.vue

```diff
@@ -36,6 +44,10 @@
       emits('update:modelValue', value)
     },
   })
+
+  const handleFocus = (): void => {
+    emits('focus')
+  }
```

Comment:
So I may have missed something with the attribute forwarding, maybe you can help me there, but, I wanted to use `p-input` instead of a native input in the combobox component. The `focus` didn't seem to be forwarded, which that component uses to `select()` all the current text when entering the field (a nice affordance). I believe I could get some sort of focus event with `ref.el.focus()`, but I wanted to be able to use the `@focus=""` binding directly.